boards/arm/bl653_dvk/doc/bl653_dvk.rst

```diff
@@ -0,0 +1,229 @@
+* BUTTON4 = SW10 = P0.25
+* RESET = SW3 = nReset/IF BOOT
+
+Programming and Debugging
+*************************
+
+Applications for the ``bl653_dvk`` board configuration can be
+built and flashed in the usual way (see :ref:`build_an_application`
+and :ref:`application_run` for more details); however, the standard
+debugging targets are not currently available.
+
+Flashing
+========
+
+Follow the instructions in the :ref:`nordic_segger` page to install
+and configure all the necessary software. Further information can be
+found in :ref:`nordic_segger_flashing`. Then build and flash
+applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Here is an example for the :ref:`hello_world` application.
+
+First, run your favorite terminal program to listen for output.
+
+NOTE: On the BL653 development board, the FTDI USB should be used to access the UART console.
+
+.. code-block:: console
+
+   $ minicom -D <tty_device> -b 115200
+
+Replace :code:`<tty_device>` with the port where the BL653 development kit
+can be found. For example, under Linux, :code:`/dev/ttyUSB0`.
+
+Then build and flash the application in the usual way.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: bl653_dvk
+   :goals: build flash
+
+Debugging
+=========
+
+Refer to the :ref:`nordic_segger` page to learn about debugging Nordic based boards with a
+Segger IC.
+
+Testing Bluetooth on the BL653 DVK
+***********************************
+Many of the Bluetooth examples will work on the BL653 DVK.
+Try them out:
+
+* :ref:`ble_peripheral`
+* :ref:`bluetooth-eddystone-sample`
+* :ref:`bluetooth-ibeacon-sample`
+
+
+Testing the LEDs and buttons on the BL653 DVK
+************************************************
+
+There are 2 samples that allow you to test that the buttons (switches) and LEDs on
+the board are working properly with Zephyr:
+
+* :ref:`blinky-sample`
+* :ref:`button-sample`
+
+You can build and flash the examples to make sure Zephyr is running correctly on
+your board. The button and LED definitions can be found in
+:zephyr_file:`boards/arm/bl653_dvk/bl653_dvk.dts`.
+
+Using UART1
+***********
+
+The following approach can be used when an application needs to use
+more than one UART for connecting peripheral devices:
+
+1. Add devicetree overlay file to the main directory of your application:
+
+   .. code-block:: console
+
+      $ cat nrf52833dk_nrf52833.overlay
+      &uart1 {
+        compatible = "nordic,nrf-uarte";
+        current-speed = <115200>;
+        status = "okay";
+        tx-pin = <14>;
+        rx-pin = <16>;
+      };
+
+   In the overlay file above, pin P0.16 is used for RX and P0.14 is used for TX
+
+2. Use the UART1 as ``device_get_binding(DT_LABEL(DT_NODELABEL(uart1)))``
+
+Overlay file naming
+===================
+
+The file has to be named ``<board>.overlay`` and placed in the app main directory to be
+picked up automatically by the build system.
```

Comment:
Maybe link to this as well: https://docs.zephyrproject.org/latest/guides/dts/howtos.html#set-devicetree-overlays
It describes the options you have for specifying overlays and where they can be located.

Reply:
Updated.